From a64a24640cb66aef9468a50be6577490dfa2a83a Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Zbigniew=20J=C4=99drzejewski-Szmek?= <zbyszek@in.waw.pl>
Date: Sat, 31 Jan 2015 23:03:25 -0500
Subject: [PATCH] core/cgroup: fix embarrassing typo

https://github.com/docker/docker/issues/10280
(cherry picked from commit a3bd89ea99214db7ca2ad6ae59181d5119a54f09)
---
 src/core/cgroup.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/core/cgroup.c b/src/core/cgroup.c
index 37db3d5289..7edb802f3f 100644
--- a/src/core/cgroup.c
+++ b/src/core/cgroup.c
@@ -382,7 +382,7 @@ void cgroup_context_apply(CGroupContext *c, CGroupControllerMask mask, const cha
                 }
         }
 
-        if ((mask & CGROUP_MEMORY) & !is_root) {
+        if ((mask & CGROUP_MEMORY) && !is_root) {
                 if (c->memory_limit != (uint64_t) -1) {
                         char buf[DECIMAL_STR_MAX(uint64_t) + 1];
 
